pages/sdk/relay-kit/guides/onchain-identifier.mdx

```diff
@@ -0,0 +1,199 @@
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Track ERC-4337 transactions with an onchain identifier
+
+In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations `callData`.
```

Comment:
```suggestion
In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations (`callData`).
```

Reply:
Should we also link to the User Operations entry in the Glossary?

Reply:
Re: my suggestion regarding the parenthesis, feel free to ignore if that's not accurate way to represent.

Reply:
Regarding the parenthesis, this should be the right option:
```
user operation's `callData`.
```